vector/src/main/java/im/vector/util/PreferencesManager.java

```diff
@@ -289,51 +287,6 @@ public static void setDidAskUserToIgnoreBatteryOptimizations(Context context) {
                 .apply();
     }
 
-    public static String getIntegrationManagerUiUrl(Context context) {
-        return PreferenceManager.getDefaultSharedPreferences(context).getString(SETTINGS_INTEGRATION_MANAGER_UI_URL,
```

Comment:
There is no migration of the previous user settings?

Reply:
It was stored before in pref but never changed (set in VectorApp at startup). Should be no need to migrate as if a custom IM is set it will be taken from account data now